src/pages/index.module.css

```diff
@@ -34,6 +34,11 @@ h2.heroTagline {
   filter: drop-shadow(2px 2px 2px #666);
 }
 
+.hero__subtitle {
+  position: relative;
+  z-index: 4;
```

Comment:
This won't work properly as it's only relative to elements in this range (?).

We can define this property in `HomepageTitle` instead.